src/feed/feed.service.ts

```
@@ -5,8 +5,13 @@ import { SavedPost } from './entities/savedpost.entity';
 import { Post } from '../post/entities/post.entity';
 import { CreateFeedDto } from './dto/create-feed.dto';
 import { UpdateFeedDto } from './dto/update-feed.dto';
+import { Job } from "../jobs/entities/job.entity"
+import { NotificationsService } from 'src/notifications/notifications.service';
+import { JobStatus } from './enums/job-status.enum';
+import { CreateCommentDto } from './dto/create-comment.dto';
+import { Comment } from './entities/comment.entity';
+import { User } from 'src/auth/entities/user.entity';
 import { Report } from './entities/report.entity';
-import { Job } from "../jobs/entities/job.entity";
 import { NotificationsService } from '../notifications/notifications.service';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NotificationsService'. Consider to delete it or rename it.

'NotificationsService' is defined here:


(lint/suspicious/noRedeclare)